sql/scan.go

```diff
@@ -401,9 +409,9 @@ func (n *scanNode) computeOrdering(columnIDs []ColumnID) []int {
 			}
 		}
 		if !found {
```

Comment:
Yes. Probably even cleaner just to initialize `ordering` to the correct size and set `ordering[j]` within the loop above.